src/System.Text.Json/tests/Serialization/CustomConverterTests.DictionaryGuidConverter.cs

```diff
@@ -0,0 +1,186 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit;
+
+namespace System.Text.Json.Serialization.Tests
+{
+    public static partial class CustomConverterTests
+    {
+        /// <summary>
+        /// Demonstrates a <see cref="Dictionary{Guid, TValue}"> converter using a JSON object with property names representing keys.
+        /// Sample JSON for <see cref="Dictionary{Guid, object}">: {"2E6E1787-1874-49BF-91F1-0F65CCB6C161":{}}
```

Comment:
Should this be TValue instead of object or am I misunderstanding the intent here?
```suggestion
        /// Sample JSON for <see cref="Dictionary{Guid, TValue}">: {"2E6E1787-1874-49BF-91F1-0F65CCB6C161":{}}
```

Reply:
Since this supports any TValue, the sample is just for an object which has {}. A sample for an integer would not have {} for example.